x-pack/plugins/observability/public/components/shared/page_template/README.md

```diff
@@ -140,3 +142,30 @@ The `<ObservabilityPageTemplate />` component is a wrapper around the `<KibanaPa
 After these two steps we should see something like the following (note the navigation on the left):
 
 ![Page template rendered example](./page_template.png)
+
+## Adding badge
+
+You can add a badge beside the label by using the property `sideBadgeLabel?: string`.
+
+```js
+setup(core: CoreSetup, plugins: PluginsSetup) {
+    plugins.observability.navigation.registerSections(
+      of([
+        {
+          label: 'A solution section',
+          sortKey: 200,
+          entries: [
+            { label: 'Backends', app: 'exampleA', path: '/example', sideBadgeLabel: 'NEW'  },
+          ],
+        }
+      ])
+    );
+  }
+
+```
+![Badge example](./badge.png)
+
+The badge is going to be shown until user clicks on the menu item for the first time. Then we'll save an information at local storage, following this pattern `observability.nav_item_badge_visible_${app}${path}`, the above example would save `observability.nav_item_badge_visible_exampleA/example`. And the badge is removed. It'll only show again if the item saved at local storage is removed or set to `false`.
+
+
```

Comment:
```suggestion
It's recommended to remove the badge (e.g. a new feature promotion) in the subsequent release.

To avoid the navigation flooding with badges, we also want to propose keeping it to maximum 2 active badges for every iteration
```

Reply:
Just wanted to add this recommendation when adding badges 👍